set_toolchain.sh

```diff
@@ -124,10 +124,13 @@ export PERFORM
 #
 case "$lmi_build_type" in
     (*-*-linux*)
-        w0="$(winepath -w "$localbindir" | sed -e's/\\/\\\\/g')"
-        w1="$(winepath -w "$locallibdir" | sed -e's/\\/\\\\/g')"
-        export WINEPATH="$w0;$w1"
-        export  PERFORM="wine"
+        if [ "x86_64-pc-linux-gnu" != "$LMI_TRIPLET" ]
```

Comment:
Let's rather do something like this:
```suggestion
        case "$LMI_TRIPLET" in
            (*-*-mingw32)
                ... use Wine ...
                ;;
        esac
```